shared/src/business/useCases/getCaseInteractor.ts

```diff
@@ -86,15 +87,20 @@ const getCaseForExternalUser = ({
  * @param {Object} caseRecord the original caseRecord
  * @returns {Object} decorated caseRecord
  */
-export const decorateForCaseStatus = (caseRecord: Case) => {
+export const decorateForCaseStatus = <T extends RawCase | SealedCase>(
+  caseRecord: T,
+) => {
   // allow document service
-  caseRecord.canAllowDocumentService =
+  (caseRecord as any).canAllowDocumentService =
     canAllowDocumentServiceForCase(caseRecord);
```

Comment:
Would you consider making a new type, adding `canAllowDocumentService` to SealedCase, or (maybe?) removing SealedCase as a type from this function?